src/components/FAQ.js

```diff
@@ -0,0 +1,129 @@
+const FAQ = () => {
+    return (    <div className="container mt-3 ">
+    <div className="accordion " id="accordionExample">
+    <h3 className="text-center">Frequently Asked Questions:</h3>
+        <div className="accordion-item">
+          <h2 className="accordion-header" id="headingOne">
+            <button className="accordion-button text-dark bg-light" type="button" data-bs-toggle="collapse" data-bs-target="#collapseOne" aria-expanded="false" aria-controls="collapseOne">
+            01. How can I order online?
+            </button>
+          </h2>
+          <div id="collapseOne" className="accordion-collapse collapse" aria-labelledby="headingOne" data-bs-parent="#accordionExample">
+            <div className="accordion-body">
+                <p>You can order directly on our website or through our mobile app. Simply browse the menu, customize your items, and check out securely online. </p>
```

Comment:
We don't have a mobile app. It would be more appropriate to say available on mobile web and desktop browsers.

Reply:
done

Reply:
👍